clients/sfizz_render.cpp

```diff
@@ -165,7 +166,7 @@ int main(int argc, char** argv)
     ERROR_IF(!synth.loadSfzFile(sfzPath), "There was an error loading the SFZ file.");
     LOG_INFO(synth.getNumRegions() << " regions in the SFZ.");
 
-    fmidi_smf_u midiFile { fmidi_smf_file_read(midiPath.u8string().c_str()) };
+    fmidi_smf_u midiFile { fmidi_smf_file_read(u8EncodedString(midiPath).c_str()) };
```

Comment:
The process becomes more complex.
expanded process:
```
const std::u8string &s = midiPath.u8string();
fmidi_smf_file_read(std::string(s.begin(), s.end()).c_str())
```
In my opinion, the conversion to std::string is waste.
I prefer just `midiPath.u8string().c_str()` but the choice is yours.

Reply:
Yeah, except we at least need to `reinterpret_cast` it to `const char*`. This would avoid one copy here but it's a minor offense, and it makes this method ubiquitous across the codebase where most other conversions are to owning `std::string`.